runtime/development/src/lib.rs

```diff
@@ -1914,6 +1914,15 @@ impl_runtime_apis! {
 		}
 	}
 
+	impl runtime_common::apis::LoansApi<Block, PoolId, Balance> for Runtime {
+		fn pool_valuation(pool_id: PoolId) -> Option<Balance>{
+			match pallet_loans::Pallet::<Runtime>::update_nav_of_pool(pool_id) {
```

Comment:
The `impl PoolNAV` is using `update_nav`, the same name es the extrinsic for `pallet_loans`. If I try to use `update_nav` here, it tries to use the extrinsic instead of the one from the `PoolNAV` trait. Maybe some  trait bound magic we could do otherwise? 

In the end, `update_nav` is calling `update_nav_of_pool` internally, so I am using this instead.